base/build.gradle.kts

```diff
@@ -61,6 +61,7 @@ dependencies {
     // DI Nav for BVM factory
     implementation("androidx.hilt:hilt-navigation-compose:1.1.0-alpha01")
     implementation("io.insert-koin:koin-androidx-compose-navigation:3.5.3")
+    implementation(project(":compose-android"))
```

Comment:
- Why is this needed? 
- This might cause issues. A user wanting only base module will have to deal with compose-android module integration as well.
- Please remove!